apps/docs/pages/guides/platform/exhaust-swap.mdx

```diff
@@ -0,0 +1,71 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'exhaust-swap',
+  title: 'High Swap Usage',
+  description:
+    'Learn what high Swap usage could mean for your Supabase instance and what could have caused it.',
+}
+
+Learn what high Swap usage could mean for your Supabase instance and what could have caused it.
+
+## What is Swap for?
+
+<Admonition type="tip">
+
+High Swap is generally not a concern if you find other resources on your instance are not constrained.
+
+</Admonition>
+
+Every Supabase project runs in its dedicated virtual machine. Your instance will have a different set of hardware provisioned depending on your [compute add-on](https://supabase.com/docs/guides/platform/compute-add-ons). Your hardware may not be suitable for the intended workload and may experience high Swap usage.
+
+Swap is a portion of your instance's disk that is reserved for the operating system to use when the available RAM has been utilized. As it uses the disk, Swap is slower to access and is generally used as a last resort.
+
+Swap can be used even if your instance has plenty of RAM. If this is the case, do not worry.
+
+Your instance might try to "preemptively swap" by swapping background processes to make space for your traffic in RAM.
+
+### When is high Swap concerning?
+
+High Swap is concerning if your instance is using all of the available RAM (i.e. consistently using more than 75%). 
+
+High Swap usage could cause noticeable performance impacts to your instance, typically this could manifest as:
+
+- **Slower query responses.** 
+- **Degraded performance due to swapping regularly between RAM and disk.**
+- **Higher Disk I/O due to swapping regularly.**
+
+## Monitor your Swap
+
+You can check your Swap usage directly on the Supabase Platform. For this go to database health in the reports section or [click here](https://supabase.com/dashboard/project/_/reports/database) and select your project.
+
+It is also possible to monitor your resources and set up alerts using Prometheus/Grafana. You can find a guide for this [here](https://supabase.com/docs/guides/platform/metrics).
+
+An [example repository](https://github.com/supabase/supabase-grafana) to ingest metrics and visualize them with Grafana is provided in the linked guide, where we maintain a [list of the exported metrics](https://github.com/supabase/supabase-grafana/blob/main/docs/metrics.md).
+
+Some useful metrics to monitor are:
+
+- `node_memory_SwapFree_bytes` - Shows the total amount of Swap available in bytes.
```

Comment:
Mention the ones for RAM utilization as well? And page fault rate? Page fault rate might not be included in older projects thougth